x/ccv/consumer/keeper/params.go

```diff
@@ -6,8 +6,7 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
 
-	"github.com/cosmos/interchain-security/v3/x/ccv/consumer/types"
-	ccvtypes "github.com/cosmos/interchain-security/v3/x/ccv/types"
+	types "github.com/cosmos/interchain-security/v3/x/ccv/types"
```

Comment:
Related to the [comment](https://github.com/cosmos/interchain-security/pull/1071/files#r1300611106). I guess in these cases we can use `ccvtypes` even in this PR since the name is already changed?

Reply:
Well the code that was previously in "github.com/cosmos/interchain-security/v3/x/ccv/consumer/types" is now in "github.com/cosmos/interchain-security/v3/x/ccv/types", so originally using the `types` alias here avoids a big diff. But I'll make the change since we're already talking about it https://github.com/cosmos/interchain-security/commit/065c2cd3ad0581aae39c8acd2ef0016ea6a44e8c